internal/http/services/owncloud/ocdav/propfind/propfind.go

```diff
@@ -64,6 +64,32 @@ const (
 	tracerName = "ocdav"
 )
 
+var (
+	audioKeys = []string{
+		"album",
+		"albumArtist",
+		"artist",
+		"bitrate",
+		"composers",
+		"copyright",
+		"disc",
+		"discCount",
+		"duration",
+		"genre",
+		"hasDrm",
+		"isVariableBitrate",
+		"title",
+		"track",
+		"trackCount",
+		"year",
+	}
```

Comment:
Can we have them all lower here too? We can still keep the `strings.ToLower` but having them camel case here will lead to confusion. If this is not possible, could you make a comment stating that they will be `ToLower`d later?

Reply:
No, we can't. We used these keys not only for the prop names but also to look up the values in the metadata storage through keys generated with `metadataKeysWithPrefix("libre.graph.audio", audioKeys)...)`
They are camel case just as in the LibreGraph facet.

Reply:
Ah I see. No need to rename. But please add a comment that they are also used as propnames all lowered. That will make later debugging/refactoring/feature adding easier.